suites/reef/cephfs/tier-2_cephfs_journal_tool.yaml

```diff
@@ -0,0 +1,119 @@
+---
+tests:
```

Comment:
As this is a feature exists in quincy too, please consider including suite in quincy folder. Also, in squid folder.